sdcm/cluster.py

```diff
@@ -2231,7 +2231,7 @@ def install_mgmt(self, package_url: Optional[str] = None) -> None:
 
         if self.distro.is_rhel_like:
             self.install_epel()
-            self.remoter.sudo("yum install python36-PyYAML -y", retry=3)
+            self.remoter.sudo("yum install python3-PyYAML -y", retry=3)
```

Comment:
Can be replaced with self.install_package

Reply:
+1